### PR TITLE
Support 'rubric' metadata to show info on a quiz's front page

### PR DESCRIPTION
**Pull request by Morpheu5**
---

**Pull Request Check List**
- [ ] Unit Tests & Regression Tests Added (Optional)
- [x] Removed Unnecessary Logs/System.Outs/Comments/TODOs
- [ ] Peer-Reviewed

### CODE REVIEW

File: src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java [hunk 1]
```diff
@@ -410,6 +410,7 @@ public PGInterval getPGInterval() {
     public static final String[] ADDRESS_FIELDNAMES = {"addressLine1", "addressLine2", "town", "county", "postalCode"};
     public static final String SEARCHABLE_CONTENT_FIELDNAME = "searchableContent";
     public static final String VISIBLE_TO_STUDENTS_FIELDNAME = "visibleToStudents";
+    public static final String QUIZ_RUBRIC_FIELDNAME = "rubric";
```
Comment (bh412) [inline src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java:413]: I'm not certain this is needed, please do correct me if I'm wrong though!

**Review by bh412 — COMMENTED** [review]
Do we need an associated front end change (at least regenerating/ adding ruberic to IsaacApiTypes) so it recognises the changes to IsaacQuizDto?

**Comment by Morpheu5** [comment]
> Do we need an associated front end change (at least regenerating/ adding ruberic to IsaacApiTypes) so it recognises the changes to IsaacQuizDto?

Did I forget to make the change? Just adding the field should do, anyway.

File: src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java (hunk 1 above)
Comment (Morpheu5) [reply src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java:413]: Not sure either but I figured it was not doing any harm.

**Comment by bh412** [comment]
You made the changes, I didn't see the other PRs, my bad! All good!

**Pull request closed without merging**